templates/openstack/features.html

```diff
@@ -13,7 +13,7 @@
       <h1>Enterprise private cloud</h1>
       <h2 class="p-heading--3">Total automation. Flexible technology choices. <br class="u-hide--small">Enterprise features. Stability and security above all.</h2>
       <p>Everything you need for your mission&ndash;critical workloads.</p>
-      <p><a href="/openstack/contact-us" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'Contact us for enterprise private cloud', 'eventLabel' : 'Get an enterprise private cloud - Top CTA', 'eventValue' : undefined });" class="p-button--positive js-invoke-modal">Build your own production-grade cloud</a></p>
+      <p><a href="/engage/openstack-deployment-guide" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'Build your own production-grade cloud', 'eventLabel' : 'Get an enterprise private cloud - Top CTA', 'eventValue' : undefined });" class="p-button--positive">Build your own production-grade cloud</a></p>
```

Comment:
```suggestion
      <p><a href="/engage/openstack-deployment-guide" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'Build your own production-grade cloud', 'eventLabel' : 'Get an enterprise private cloud - engage page', 'eventValue' : undefined });" class="p-button--positive">Build your own production-grade cloud</a></p>
```
I wonder if this should specify engage page? As this is not a top CTA lead

Reply:
yeah I don't know, up to Growth Marketing @laszlokajtar any thoughts? Do we need to change this?